From 9260bedc9b0ee043a106b173d28ddfc5112d698b Mon Sep 17 00:00:00 2001
From: Daniel Kahn Gillmor <dkg@fifthhorseman.net>
Date: Fri, 30 Aug 2024 00:22:14 -0400
Subject: [PATCH 5/5] Relax dependencies

The test suites all pass with these dependencies relaxed.

I've reported this flexibility upstream at
https://github.com/rpgp/rpgp/pull/396
---
 Cargo.toml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Cargo.toml b/Cargo.toml
index dff28a5..8562045 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -54,7 +54,7 @@ version = "0.10.3"
 version = "0.5"
 
 [dependencies.base64]
-version = "^0.22.0"
+version = "0.21.7"
 
 [dependencies.bitfield]
 version = "0.14"
@@ -199,7 +199,7 @@ package = "num-bigint-dig"
 version = "0.2.6"
 
 [dependencies.num_enum]
-version = "0.7"
+version = "0.5.7"
 
 [dependencies.ocb3]
 version = "0.1"
-- 
2.45.2

